alcs-frontend/src/app/shared/drag-drop-board/drag-drop-board.component.ts

```diff
@@ -32,6 +35,14 @@ export class DragDropBoardComponent {
       return;
     }
 
+    debugger;
+    if (targetColumn.status === 'RELE' && selectedCard.paused) {
+      this.toastService.showErrorToast(
+        "Paused cards cannot be moved to the 'Decision Released' column. Please unpause the card first and try again"
```

Comment:
is there any enum that contains description "Decision Released" ?

Reply:
I could add a subscriber to the applicationStatus from the service, look up the value from there using the code, and use that label?